README.md

```diff
@@ -39,6 +39,8 @@ This repo is only available to Igalia members. If you have access to the relevan
  - `third_party/OVRPlatformSDK/` for Oculus (should contain a `Android` and `include` folders)
  - `third_party/hvr/` for Huawei (should contain  `arm64-v8a`, `armeabi-v7a` and `include` folders)
  - `third_party/wavesdk/` for Vive (should contain a `build` folder, among other things)
+ - `third_party/picooxr` [Pico OpenXR Mobile SDK](https://developer-global.pico-interactive.com/sdk?deviceId=1&platformId=3&itemId=11) (should contain a `OpenXR` folder)
```

Comment:
I'll comment about the third_party stuff later, but we'll have to change this clarification)